src/NuGet.Core/NuGet.Commands/RestoreCommand/RequestFactory/ProjectJsonRestoreRequestProvider.cs

```diff
@@ -69,7 +69,7 @@ public Task<bool> Supports(string path)
             // Get settings relative to the input file
             var settings = restoreContext.GetSettings(file.DirectoryName);
 
-            var sources = restoreContext.GetEffectiveSources(settings);
+            var sources = restoreContext.GetEffectiveSources(settings, null);
```

Comment:
Yeah I have a task that will remove this before the "big" merge. 
This will make some tests fail too. 
So it will be fixed within that work.